windows/src/desktop/kmshell/main/initprog.pas

```diff
@@ -155,6 +159,14 @@ implementation
 function FirstRun(FQuery, FDisablePackages, FDefaultUILanguage: string): Boolean; forward;  // I2562
 procedure ShowKeyboardWelcome(PackageName: WideString); forward;  // I2569
 procedure PrintKeyboard(KeyboardName: WideString); forward;  // I2329
+{
+  Calls the Windows shell to execute the file passed in `SavePath`
+  @param  SavePath  The full path file name to be open by the shell
+}
+procedure ExecuteInstall(SavePath: String); forward;
+{
+  Starts the install of the cached files
+}
```

Comment:
What does this comment refer to? `Main`? Then the empty line should be removed or put above the comment.

Reply:
Dead comment I will remove